engine/src/hopeit/dataobjects/payload.py

```diff
@@ -0,0 +1,143 @@
+"""
+Payload tools to serialized and deserialze data objects
```

Comment:
"to serialize"

instead of just "data objects" put "event payloads and responses, including dataobjects"